index.html

```diff
@@ -26,8 +25,163 @@ <h1>  Hey there. I’m Mariane <span class="green" >I’m a software developer</
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article> 
+                <article class="project">
+                    <div class ="image-placeholder"></div>
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article>
+                 <article class="project">
+                    <div class ="image-placeholder"></div>
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article>
+            </div>
+            </section>
+            <section>
+                <div class="about2">
+                    <h2>About me</h2>
+
+                <div class="about">
+                </div>
+                <div class="about-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+                </div>
+            </div>
+                <div class="butt resume-container"><button class="btn btn-resume" type="button"> Get My Resume</button></div>
+                
+                <div class="skills-container">
+                <article class="project2">
+                <div class="language-icon"> <h><img src="/images/icon-languages (1).png" alt=""></h></div>
+                <div class="lang-text">Languages </div>
+                    <ul>
+                    <div class="lang-text-list">
+                        <li  class="li-lang">Javascript</li>
+                        <li class="li-lang">Ruby</li>
+                        <li class="li-lang">html</li>
+                        <li class="li-lang">css</li>
+                    </div>
+                    </ul>
+
+                </article>
+                <article class="project2">
+                    <div class="language-icon"> <h><img src="/images/icon-frameworks (1).png" alt=""></h></div>
```

Comment:
- [ ] Good job so far 👏 As suggested by the previous reviewer, try to fix the issue of the broken images in your design because they are not visible and as far as I know, this is due to the wrong path used to reference each among those broken images. 
![image](https://user-images.githubusercontent.com/90222110/192060951-12dab64c-3ae3-47c2-8ebb-8d21a0af794d.png)
![image](https://user-images.githubusercontent.com/90222110/192061059-ca0122a8-a433-4679-8846-12ff51ffd570.png)

Please check them and if there is any fault, please correct it 😃